server/app/services/CfJsonDocumentContext.java

```diff
@@ -405,11 +418,40 @@ public Optional<Double> readDouble(Path path) {
 
   /**
    * Attempt to read a list at the given {@link Path}. Returns {@code Optional#empty} if the path
-   * does not exist or a value other than an {@link ImmutableList} of longs is found.
+   * does not exist or a value other than an {@link ImmutableList} of Longs is found.
+   *
+   * @param path the {@link Path} to the list
+   * @return an Optional containing an ImmutableList<Long>
+   */
+  public Optional<ImmutableList<Long>> readLongList(Path path) {
+    TypeRef<ImmutableList<Long>> typeRef = new TypeRef<ImmutableList<Long>>() {};
```

Comment:
I think this can be `TypeRef<ImmutableList<Long>> typeRef = new TypeRef<>() {};`, ditto below. Let me know if wrong.

Reply:
I've inlined the both.